src/components/CalendarCell/CalendarCell.st.css

```diff
@@ -11,7 +11,9 @@
 /*Defaults*/
 :vars {
     DefaultBorderWidth: 1px;
+    DefaultPastFontColor: color-4;
     DefaultFontColor: color-5;
+    DefaultFutureColor: color-4;
```

Comment:
is this the text color or BG?

Reply:
it's font. it's doesn't clear, you right, i update